src/main/scala/ProgramROM.scala

```diff
@@ -10,32 +10,156 @@ class ProgramROMBundle extends Bundle {
 class ProgramROM extends Module {
   val io = IO(new ProgramROMBundle)
   val content = VecInit(Array(
-    "h00004fb7".U(32.W), // li t6, 0x4000
-    "h000f8f93".U(32.W),
-    "h01200513".U(32.W), // li a0, 0x12
-    "h00afa023".U(32.W), // sw a0, 0(t6)
-    "h00000513".U(32.W), // li a0, 0x0
-    "h00afa223".U(32.W), // sw a0, 4(t6)
-
-    "h80020f37".U(32.W), // li t5, 0x80020000
-    "h00100293".U(32.W), // li t0, 1
-    "h00200313".U(32.W), // li t1, 2
-    "h006303b3".U(32.W), // add t2, t1, t1
-    "h005383b3".U(32.W), // add t2, t2, t0
-    "h10012e37".U(32.W), // li t3, 0x10012000
-    "h000e0e13".U(32.W),
-    "h007e2023".U(32.W), // sw t2, 0(t3)
-    // label
-    "h007f2223".U(32.W), // sw t2, 4(t5)
-    "h004f2e03".U(32.W), // lw t3, 4(t5)
-    "h01c38eb3".U(32.W), // add t4, t2, t3
-    "h007eeeb3".U(32.W), // or t4, t4, t2
-    "h10012e37".U(32.W), // li t3, 0x10012000
-    "h000e0e13".U(32.W),
-    "h01de2023".U(32.W), // sw t4, 0(t3)
-    "h00138393".U(32.W), // addi t2, t2, 1
-    // j label
-    "hfe1ff06f".U(32.W)))
+//<_start>:
+"h00021197".U(32.W),    //auipc x3,0x21
```

Comment:
The new test is SO LARGE, which made it hard to understand.
But IMO we should not do intergration test in this f**king way anyhow, we can keep it like this for now, and find a more data-driven way to do it.